crates/meroctl/src/cli.rs

```diff
@@ -70,19 +78,29 @@ pub struct RootArgs {
     #[arg(env = "CALIMERO_HOME", hide_env_values = true)]
     pub home: Utf8PathBuf,
 
-    /// Name of node
-    #[arg(short, long, value_name = "NAME")]
-    pub node_name: String,
+    /// API endpoint URL
+    #[arg(long, value_name = "URL", conflicts_with = "node_name")]
+    pub api: Option<Url>,
+
+    /// Use a pre-configured node alias
+    #[arg(long, value_name = "ALIAS", conflicts_with_all = &["node_name", "api"])]
+    pub node: Option<String>,
```

Comment:
there no longer is `node_name`:

```suggestion
    /// API endpoint URL
    #[arg(long, value_name = "URL")]
    pub api: Option<Url>,

    /// Use a pre-configured node alias
    #[arg(long, value_name = "ALIAS", conflicts_with = "api")]
    pub node: Option<String>,
```